server/migrate-mongo-config.js

```diff
@@ -0,0 +1,30 @@
+// In this file you can configure migrate-mongo
+
+const config = {
+  mongodb: {
+    // TODO: Update this with prod config
+    url: "mongodb://localhost:27017",
+
+    // TODO: Update this with prod db name
+    databaseName: "ofi-testdata",
+
+    options: {
+      useNewUrlParser: true, // removes a deprecation warning when connecting
+      useUnifiedTopology: true, // removes a deprecating warning when connecting
+      //   connectTimeoutMS: 3600000, // increase connection timeout to 1 hour
```

Comment:
We could get rid of these commented out config params right?

Reply:
bump on this comment!

Reply:
My bad, missed these comments, will take care of them now